### ISSUE TITLE
Possible bug: running as admin, drag&drop of APK file seems possible but it's not

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Genymobile/scrcpy#1496]
- [V ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [V ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [e.g. Debian, Windows, macOS...]
Windows
 - scrcpy version: [e.g. 1.12.1]
v1.14
 - installation method: [e.g. manual build, apt, snap, brew, Windows release...]
Extract from zip
 - device model:
Pixel 4 and Galaxy Note 8
 - Android version: [e.g. 10]
Android R and 10.

**Describe the bug**
Run as admin.
Try to drag some APK to the window.

As I've read, Windows OS doesn't allow to do such a thing from an app that wasn't run as admin.
And here it also doesn't work.

However, the dragging seems like it does allow it.
If somehow you do get it, please fix it. If not, I suggest to change the icon.

**Comment by rom1v** [comment]
Please post the content of your terminal after you attempted to drag&drop an APK.

**Comment by AndroidDeveloperLB** [comment]
Nothing is written to the console. See :

[2020-06-11_15-39-45.zip](https://github.com/Genymobile/scrcpy/files/4764779/2020-06-11_15-39-45.zip)

**Comment by AndroidDeveloperLB** [comment]
But really, if you can somehow overcome it, it could be great.
I suggest not to focus on it and just block it though. According to many places I've read, it's impossible.

**Comment by rom1v** [comment]
OK, it seems that the drag&drop event is not even notified by SDL.

If you do the same thing as non-admin, do you have adb push logs in the console?

**Comment by AndroidDeveloperLB** [comment]
When I do it without admin, it works fine:

```
INFO: Request to install C:\Users\User\Desktop\tcandroid300-universal.apk
INFO: Installing C:\Users\User\Desktop\tcandroid300-universal.apk...
Performing Streamed Install
Success
INFO: C:\Users\User\Desktop\tcandroid300-universal.apk successfully installed
```

**Comment by rom1v** [comment]
OK, so it means that the window manager does not even send the drag&drop event to scrcpy. If that's the case, there is nothing scrcpy can do I guess.

**Comment by AndroidDeveloperLB** [comment]
Right. However, I've noticed that on other apps (Chrome, Android Studio, Android emulator), it shows an icon that this action is impossible.
On Scrcpy, it shows as if it's ok.
If you think like me that it's probably impossible to handle it, I suggest you to use the same behavior : show indicator that it's impossible.
